src/annotator/guest.js

```diff
@@ -116,11 +116,8 @@ export default class Guest {
     this.element = element;
     this._emitter = eventBus.createEmitter();
     this.visibleHighlights = false;
-    this.adderToolbar = document.createElement('hypothesis-adder');
-    this.adderToolbar.style.display = 'none';
-    this.element.appendChild(this.adderToolbar);
 
-    this.adderCtrl = new Adder(this.adderToolbar, {
+    this.adderCtrl = new Adder(this.element, {
```

Comment:
I suggest renaming `this.adderCtrl` to `this.adder` since there is no longer a need to differentiate between two different things related to the adder.

Reply:
I thought the same. Thanks, I will do it.

Reply:
I also renamed `adderCtrl` to `adder` in `adder-test.js`.